<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="yuri-PC" name="com.zixi.testing.BroadcasterUdpInStreamCreationTest" tests="5" failures="0" timestamp="10 Aug 2016 12:47:21 GMT" time="4.284" errors="1">
  <testcase name="broadcasterUdpInCreation" time="0.837" classname="com.zixi.testing.BroadcasterUdpInStreamCreationTest"/>
  <testcase name="broadcasterUdpInCreation" time="0.846" classname="com.zixi.testing.BroadcasterUdpInStreamCreationTest">
    <error type="org.json.JSONException" message="A JSONObject text must begin with &#039;{&#039; at 1 [character 2 line 1]">
      <![CDATA[org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
	at org.json.JSONObject.<init>(JSONObject.java:197)
	at org.json.JSONObject.<init>(JSONObject.java:324)
	at com.zixi.tools.JsonParser.getBroadcasterVersion(JsonParser.java:198)
	at com.zixi.drivers.ProductAboutDriver.getBroadcasterVersion(ProductAboutDriver.java:12)
	at com.zixi.testing.BroadcasterUdpInStreamCreationTest.broadcasterUdpInCreation(BroadcasterUdpInStreamCreationTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- broadcasterUdpInCreation -->
  <testcase name="broadcasterUdpInCreation" time="0.850" classname="com.zixi.testing.BroadcasterUdpInStreamCreationTest"/>
  <testcase name="broadcasterUdpInCreation" time="0.865" classname="com.zixi.testing.BroadcasterUdpInStreamCreationTest"/>
  <testcase name="broadcasterUdpInCreation" time="0.886" classname="com.zixi.testing.BroadcasterUdpInStreamCreationTest"/>
</testsuite> <!-- com.zixi.testing.BroadcasterUdpInStreamCreationTest -->
